bodhi/server/services/updates.py

```diff
@@ -536,6 +538,23 @@ def new_update(request):
 
             if len(releases) > 1:
                 result = dict(updates=updates)
+
+            if from_tag:
+                koji = buildsys.get_session()
+                # Validate that <koji_tag>-pending-signing and <koji-tag>-testing exists,
+                # if not create them
+                koji_tag_pending_signing = from_tag + "-pending-signing"
+                koji_tag_testing = from_tag + "-testing"
```

Comment:
I'm not sure if we want to hardcode "-pending-signing" and "-testing" or put it in configuration.

@cverna, @abompard?